docroot/themes/humsci/humsci_basic/src/scss/components/_buttons.scss

```
@@ -65,6 +71,12 @@ button.more-link,
   .hb-dark-inversion .views-element-container & {
     @include hb-global-color('color', 'white');
   }
+
+  @include hb-traditional {
+    .ht-pairing-warbler &:hover, &:focus {
```

Maybe I'm misreading, but should this be: 

```suggestion
    .ht-pairing-warbler &:hover, 
    .ht-pairing-warbler  &:focus {
```

Perhaps it compiles differently than I would expect?